code/modules/projectiles/projectile/ego_bullets/zayin.dm

```diff
@@ -57,3 +57,12 @@
 	name = "oceanic"
 	damage = 11		//Worse than tough lol
 	damage_type = WHITE_DAMAGE
+
+/obj/projectile/ego_bullet/ego_dud
+	name = "dud"
+	damage = 1
+	damage_type = RED_DAMAGE
+
+/obj/projectile/ego_bullet/ego_dud/Initialize()
+	qdel(src)
+	..()
```

Comment:
NEVER do this with Initialize() man
eighter do `. = ..()` or `return ..()` as otherwise we get silent errors from `..()`
```suggestion
	return ..()
```

Reply:
Gotcha. Thankfully, this bullet was never actually implemented due to it breaking the guns it uses somehow, but I'll try out the suggested code to see if it works.

Reply:
Tested both suggestions, neither seem to actually fix the "permanently broken gun" problem. Gonna just scrap the "dud round" idea for Death dealer until I'm more experienced with BYOND witchcraft.